tcl/org.testeditor.tcl.dsl.ui.tests/src/org/testeditor/tcl/dsl/ui/tests/editor/DropTargetListenerUpdateModelTest.xtend

```diff
@@ -0,0 +1,441 @@
+package org.testeditor.tcl.dsl.ui.tests.editor
+
+import com.google.common.base.Strings
+import java.nio.file.Files
+import java.nio.file.Paths
+import javax.inject.Inject
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.serializer.ISerializer
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.aml.AmlModel
+import org.testeditor.aml.Component
+import org.testeditor.aml.ComponentElement
+import org.testeditor.dsl.common.testing.AbstractTest
+import org.testeditor.dsl.common.testing.DslParseHelper
+import org.testeditor.tcl.ComponentTestStepContext
+import org.testeditor.tcl.TclModel
+import org.testeditor.tcl.dsl.TclStandaloneSetup
+import org.testeditor.tcl.dsl.ui.editor.DropTargetXtextEditorListener
+import com.google.common.io.ByteStreams
+
+class DropTargetListenerUpdateModelTest extends AbstractTest {
```

Comment:
The given is about 25 line of code. The when is the inserted code. The when is 25 lines of code + the inserted code. This hardly fits on a screen and is difficult to read because you have to compare the given and the then line by line to understand the different. This was the result of the review with Gunther. To make the test readable I created an easy to read format where I combined the given and the when. 
To extract the methods to a helper class is a good idea but I did not find an appropriate class to place my code.